socialhome/streams/app/components/StreamElement.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div>
+        <p v-html="post.content"></p>
+        <div class="grid-item-author-bar mt-1">
+            <div class="profile-box-trigger">
```

Comment:
> Vue to watch for a click even on this div instead and then change some component state to make `profile-box` below visible. 

I know. This PR is not finished at all. I first rewrite the template before taking care of the events ;)